x/stake/keeper/validator.go

```diff
@@ -261,6 +261,13 @@ func (k Keeper) GetLastValidators(ctx sdk.Context) (validators []types.Validator
 	return validators[:i] // trim
 }
 
+// returns an iterator for the consensus validators in the last block
+func (k Keeper) LastValidatorsIterator(ctx sdk.Context) (iterator sdk.Iterator) {
+	store := ctx.KVStore(k.storeKey)
+	iterator = sdk.KVStorePrefixIterator(store, LastValidatorPowerKey)
+	return iterator
```

Comment:
Why even bother with this function? we should just add the two lines to `IterateLastValidators`

Reply:
Did it for the reason on the below comment because I originally did the `PowerStoreInvariant` check using Last Validators rather than current.  Switched it to current, but left this function there, to make it "consistent" with the `ValidatorsPowerStoreIterator`.